lk2nd/device/dts/msm8974/samsung.dts

```diff
@@ -38,6 +38,32 @@
 		};
 	};
 
+	hltetmo {
+		model = "Samsung Galaxy Note 3 (SM-N900T)";
+		compatible = "samsung,hlte";
+		qcom,msm-id = <0x7E01FF01 7 0>;
+		lk2nd,match-bootloader = "N900T*";
+
+		lk2nd,dtb-files = "msm8974-samsung-hlte";
+
+		gpio-keys {
+			compatible = "gpio-keys";
+			home {
+				lk2nd,code = <KEY_HOME>;
+				gpios = <&pmic 3 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+			};
+			down {
+				lk2nd,code = <KEY_VOLUMEDOWN>;
+				gpios = <&pmic 2 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+			};
+
+			up {
+				lk2nd,code = <KEY_VOLUMEUP>;
+				gpios = <&pmic 5 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+			};
```

Comment:
```suggestion
			compatible = "gpio-keys";

			button-home {
				lk2nd,code = <KEY_HOME>;
				gpios = <&pmic 3 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
			};

			button-volume-down {
				lk2nd,code = <KEY_VOLUMEDOWN>;
				gpios = <&pmic 2 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
			};

			button-volume-up {
				lk2nd,code = <KEY_VOLUMEUP>;
				gpios = <&pmic 5 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
			};
```

Reply:
I don't think we follow upstream linux node naming rules to the letter (so just having up/down is fine I think) but I'd indeed prefer having newlines consistent with upstream linux rules if possible. Here it's a bit mixed with some nodes having a line inbetween and some not...

Reply:
If we make any changes to my additions related to style/naming/formatting, should I just make another commit fixing all of the device entries in this file?  Otherwise it will be inconsistent even though my changes are duplicates of the existing device entries.